azure-iot-device/azure/iot/device/iothub/transport/constant.py

```diff
@@ -3,10 +3,12 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
-"""This module contains constants realted to the pipeline package.
+"""This module contains constants realted to the transport package.
```

Comment:
These cannot be enabled/disabled separately from what I understand, due to limitations of IoTHub